packages/jellyfish-wallet/src/wallet.ts

```diff
@@ -24,7 +24,7 @@ export class JellyfishWallet<Account extends WalletAccount, HdNode extends Walle
    * @return Promise<WalletAccount>
    */
   get (account: number): Account {
-    const path = `${account}/0/0`
+    const path = `1129/${account}/0/0`
```

Comment:
hmm.. not quite follow wallet stuff
wondering is this notation correct?
```{purpose}/{account}/{coint}/{index}```

Reply:
Irrelevant, this is bitcoin hdnode specific

Reply:
1. `1129` is from https://github.com/satoshilabs/slips/blob/master/slip-0044.md
2. account denoting account
3. 0 donating `internal/external` (change address)
4. address within account

But yeah, we are off track from default to make it easier to implement encrypted wallet.